include/psa/crypto_struct.h

```diff
@@ -93,10 +93,10 @@ struct psa_hash_operation_s
 };
 
 #define PSA_HASH_OPERATION_INIT {0, {0}}
-static inline struct psa_hash_operation_s psa_hash_operation_init( void )
+static inline struct psa_hash_operation_s psa_hash_operation_init(void)
```

Comment:
Not a blocker, but the commit “Fix paranthesis spacing in crypto_struct.h” (sic) shouldn't be here. `crypto_struct.h` is implementation-specific so it was intended to have Mbed TLS whitespace style.

Reply:
I did that change to address one of your comments. The spacing was mixed and I didn't see  any indication of the intended whitespace style and that is why I aligned it with the style I saw in `crypto.h`.

I will change it.